src/main/java/org/frankframework/frankdoc/model/FrankDocModel.java

```diff
@@ -313,7 +313,7 @@ public FrankElement findFrankElement(String fullName) {
 				// discovers type mismatches.
 				attribute.typeCheckDefaultValue();
 			} catch(FrankDocException e) {
-				log.warn("Attribute [{}] has an invalid default value, [{}, detail {}]", attribute.toString(), attribute.getDefaultValue(), e.getMessage());
+				log.error("Attribute [{}] has an invalid default value, [{}, detail {}]", attribute.toString(), attribute.getDefaultValue(), e.getMessage());
```

Comment:
Dit lijkt me geen reden om de build te laten falen. Default values zijn vaak beschrijvend

Reply:
Een default value kan ook een lap text zijn zoals Gerrit hier aangeeft. Een voorbeeld is de text 'application default' dwz, dat het gedrag naast een attribuut, ook middels een property instelbaar is, en dan voor de gehele applicatie geldt.

Reply:
Ik snap het. Dit is dus niet altijd letterlijk de default value, maar kan ook beschrijven wat er gebeurt als het attribute niet gezet wordt.

Reply:
Als dit zo is, moet het dan wel een warning blijven? We kunnen ook verder naar beneden, bijvoorbeeld "INFO" of "TRACE".

Reply:
Ik heb hem teruggezet op warning.